universal_transfer_operator/transfers/universal_transfer_operator.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from typing import Any
+
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+from transfers.constants import TransferMode
+from transfers.data_providers import cleanup, create_dataprovider
+from transfers.datasets.base import UniversalDataset as Dataset
+from transfers.integrations import get_transfer_integration
+from transfers.utils import check_if_connection_exists
+
+from astro.constants import LoadExistStrategy
```

Comment:
Let's not depend/import from Astro SDK

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/831d627d1a86acab300968fc56d11bc7f016b4b0)